### PR TITLE
Javascript/javascript1/week4

### PR DESCRIPTION
**Pull request by WENCHO22**

### CODE REVIEW

File: javascript/javascript1/week4/index.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    }
+
+    if (command === "What day is it today?") {
+        let todaysDate = new Date();
+        let options = {
+            dateStyle: "long",
+            /*year: "numeric",
+            month: "long",
+            day: "numeric",
+*/
```
Comment (cameronjdavis) [inline javascript/javascript1/week4/index.js]: Git provides a complete record of our files so you can safely delete commented code. We can always get it back with Git if we need to.

File: javascript/javascript1/week4/index.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
+let yourName = "";
+const toDosList = [];
+function getReply(command) {
+
+    if (command === "Hello my name is Benjamin" && !yourName) {
+        yourName += "Benjamin";
+        return "Nice to meet you " + yourName;
+    } else if (command === "Hello my name is Benjamin" && yourName) {
+        return "Yeah I know.. Nice to see you again by the way";
+    }
```
Comment (cameronjdavis) [inline javascript/javascript1/week4/index.js]: Can you change this code to handle a different name? I want to say "Hello my name is Cameron", or use another name.

File: javascript/javascript1/week4/index.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+    if (command === "Remove fishing from my todo") {
+        let fishingIndex = toDosList.indexOf("Fishing");
+        if (fishingIndex < 0) {
+            return "I couldn´t find 'fishing' in your todo list";
+        } else {
+            toDosList.splice(fishingIndex, 1);
+            return "Removed fishing from my todo";
+
```
Comment (cameronjdavis) [inline javascript/javascript1/week4/index.js:30]: Consistent formatting is important. This blank line is not needed.
```suggestion
```

**Review by cameronjdavis — CHANGES_REQUESTED** [review]

File: javascript/javascript1/week4/index.js (hunk 2 above)
Comment (WENCHO22) [reply javascript/javascript1/week4/index.js]: I just implemented some changes so it handles different names. Should I merge it now as it is?

File: javascript/javascript1/week4/index.js [hunk 4]
```diff
@@ -0,0 +1,69 @@
+let yourName = "";
+const toDosList = [];
+function getReply(command) {
+    if (command.includes("Hello my name is") && !yourName) {
+        yourName += command.replace("Hello my name is ","");
+        return "Nice to meet you " + yourName;
+    } else if (command === "Hello my name is Benjamin" && yourName) {
```
Comment (cameronjdavis) [inline javascript/javascript1/week4/index.js]: You need to check `yourName`, not `Benjamin`.

**Review by cameronjdavis — CHANGES_REQUESTED** [review]

File: javascript/javascript1/week4/index.js (hunk 2 above)
Comment (cameronjdavis) [reply javascript/javascript1/week4/index.js]: I made one more suggestion about the name Benjamin. Once that is done, you can merge it.

**Pull request merged**